samples/watchos/src/watchosMain/kotlin/main.kt

```diff
@@ -0,0 +1,73 @@
+/*
+* Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+* that can be found in the license/LICENSE.txt file.
+*/
+
+package sample.watchos
+
+import kotlinx.cinterop.*
+import platform.Foundation.*
+import platform.WatchKit.*
+import platform.darwin.NSObject
+
+// Standard entry point for WatchKit applications.
+@SymbolName("WKExtensionMain")
+external fun WKExtensionMain(argc: Int, argv: CValues<CPointerVar<ByteVar>>)
```

Comment:
Not sure if `CValues<CPointerVar<ByteVar>>` actually works in `@SymbolName(...) external fun` as intended here.

Reply:
Maybe, just fixed signature for now.